modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractGoCodegen.java

```diff
@@ -121,9 +121,9 @@ public AbstractGoCodegen() {
         typeMapping.put("date", "string");
         typeMapping.put("DateTime", "time.Time");
         typeMapping.put("password", "string");
-        typeMapping.put("File", "*os.File");
-        typeMapping.put("file", "*os.File");
-        typeMapping.put("binary", "*os.File");
+        typeMapping.put("File", "os.File");
+        typeMapping.put("file", "os.File");
+        typeMapping.put("binary", "os.File");
```

Comment:
@icubbon shall we keep the mapping (pointer to os.File) and fix the code to properly return a pointer to os.File instead?

Reply:
cc @DouglasDwyer (the author of https://github.com/OpenAPITools/openapi-generator/pull/12685)

Reply:
I'm afraid we need to revert this change and instead fix the endpoint to properly return a `*os.File` instead.

Reply:
I can take a stab at fixing the mapping to be a `*os.File`. It makes sense to have it be an `*os.File` as with the newer version of the generator, complex items are now pointers.

Reply:
@wing328 Please take a look at https://github.com/OpenAPITools/openapi-generator/pull/14340. This reverts the mapping to `*os.File` and keeps us from having any goofy `**os.File` references.